### DISCUSSION TITLE
Launching programs in Rofi with command flags

### DISCUSSION THREAD

**Discussion by m147** [i3/i3#5862] [Q&A] [answered]
I've got Simplenote-electron-bin installed but for some stupid reason it does not launch, gives the error `GPU process isn't usable. Goodbye. ` A workaround is to launch it with the flag `--in-process-gpu`
Launching it from the command line is no problem because I can just use a simple alias.  My question is, how can I have Rofi launch Simplenote with that flag?
Thanks

**Comment by okraits** [comment]
You simply type in the complete command including the flag and press `Return`.

**Reply by m147** [reply]
I tried that, doesn't work.
Also, is there any way of simplifying it, I'd rather not have to type out the whole command everytime

**Reply by okraits** [reply]
Strange, something like `alacritty -T foobar` works here.
You can also place a script in `PATH` which executes the same command.

**Reply by m147** [reply]
Yes, it is strange because I use it all time with alacritty to launch a drop down terminal.  But for some reason with simplenote it doesn't work.
I'll try setting it in the `PATH`

**Comment by mikeredev** [comment, marked as answer]
Does it have a `.desktop` entry in your `$HOME/.local/share/applications` directory? You can edit the `Exec` command. You could try creating one if not, Rofi picks from there.

**Reply by m147** [reply]
There is no entry but would I be able to add it manually?  I tried to, but I'm not sure how to format the file.

**Reply by m147** [reply]
Alright, I figured it out, just the formatting was off.  Now it is working. Thank you.
